### PR TITLE
Fix for issue #70

### PR DESCRIPTION
**Pull request by mehulj94**
Radix Sort in python

### CODE REVIEW

File: Sorting/Python/Radix Sort/radixsort.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        aList[a] = i
+        a += 1
+
+    placement *= RADIX
+
+  print aList
+
+def main():
+  a = [18,5,100,3,1,19,6,0,7,4,2]
+  radixsort(a)
```
Comment (t2013anurag) [inline Sorting/Python/Radix Sort/radixsort.py]: can you add a new line?

File: Sorting/Python/Radix Sort/radixsort.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+ def radixsort( aList ):
+  RADIX = 10
+  maxLength = False
+  tmp , placement = -1, 1
```
Comment (t2013anurag) [inline Sorting/Python/Radix Sort/radixsort.py]: remove space after tmp

File: Sorting/Python/Radix Sort/radixsort.py (hunk 1 above)
Comment (mehulj94) [reply Sorting/Python/Radix Sort/radixsort.py]: New line for?
I added a call to main. Please review.

**Comment by mehulj94** [comment]
Added call to main function

**Pull request merged**